lib/open-uri.rb

```diff
@@ -104,6 +104,7 @@ module OpenURI
     :ftp_active_mode => false,
     :redirect => true,
     :encoding => nil,
+    :max_redirects => nil,
```

Comment:
I think we should set a maximum number of redirects by default to avoid infinite redirect loops.  Maybe 64?